recipes/coremltools/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "coremltools" %}
+{% set version = "4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # The sdist on PyPI lacks at least
+  #   - LICENSE.txt
+  #   - CMakeLists.txt
+  # so we use GitHub releases.
+  url: https://github.com/apple/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: "b254b3b1bdb0d36ea83b34993d5de6d5ca3ab89a93e759d49e7a62307cad694c"
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - numpy
+  host:
+    - python
+    - numpy >=1.14.5
```

Comment:
Can you omit the constraint so it uses the `numpy` from the pinnings file (1.16)?